unit/java/io/github/azagniotov/stubby4j/yaml/YAMLMergerTest.java

```diff
@@ -0,0 +1,42 @@
+package io.github.azagniotov.stubby4j.yaml;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import static com.google.common.truth.Truth.assertThat;
+import static java.nio.file.Files.exists;
+
+/**
+ * @author Abhijit Sarkar
+ */
+public class YAMLMergerTest {
+    @Test
+    public void testMerge() throws IOException, URISyntaxException {
+        Path root = Paths.get(getClass().getResource("/").toURI());
+
+        // IntelliJ and Gradle have different output directories
+        Path resources = root.resolveSibling("resources");
+        if (!exists(resources)) {
+            resources = root.getParent().resolveSibling("resources");
+        }
+        assertThat(exists(resources)).isTrue();
+
+        File merged = YAMLMerger.merge(resources.toFile());
+        assertThat(merged).isNotNull();
+        assertThat(merged.exists() && merged.isFile() && merged.canRead()).isTrue();
```

Comment:
Can you please break the `boolean && boolean && boolean` up to three `assertThat` statements? If something fails, it is less clear what exactly failed